### ISSUE TITLE
Workflow transition: Access level change

### ISSUE THREAD

**Issue by hytsch** [joomla/joomla-cms#39510]
### Is your feature request related to a problem? Please describe.
When configuring a workflow transition, the only thing I can define is the status of the post. Why not also the access level? This would make it possible to pass a contribution from one group to the next within an editorial team.

### Describe the solution you'd like
Extension of the configuration options.

**Comment by brianteeman** [comment]
access level = viewing access level
permissions = who can create/edit/delete

you should be using the permissions for this not access and you can set the permissions for each stage

**Comment by hytsch** [comment]
Correct, and would it be grandiose to be able to define the access level in the workflow design as well when moving from one phase to the next. => Only then is a full-blown workflow possible, in which a contribution is passed on from one level (e.g. editor) to the next (e.g. reviewer).

**Comment by bembelimen** [comment]
The idea is, that you can easily (let) build a small plugin doing exactly that for you.
